### ISSUE TITLE
[Travis] Configure regular Travis build against different JDK versions

### ISSUE THREAD

**Issue by AkioFaris** [jdi-testing/jdi-light#2057]
Implement after #2038 is done

- in Travis settings configure regular builds
- create an endless PR for comments with allure report links
- modify reports.sh ang travis.yml in the branch according to the task
- send run results to Teams channel

**Comment by clockworkbanana** [comment]
**The challenges:**
1. We need to set up cron- and regular- builds in different way through the same .yaml file
2. Our current implementation of test and report stages requires a Pull Request to store the build results as comments
3. Seems that there is no possibility to cron-trigger pull_request builds

**Possible solution:**
- Triggering builds against master by cron seems to be a reasonable starting point.
- Type of event that treiggered the build may be used as condition and is also accessible as environment variable. We can use **type** condition in .travis.yml file and **TRAVIS_EVENT_TYPE** in bash scripts. See: [Conditional builds](https://docs.travis-ci.com/user/conditional-builds-stages-jobs/), [Conditions](https://docs.travis-ci.com/user/conditions-v1), [Environment variables](https://docs.travis-ci.com/user/environment-variables/)
- We use TRAVIS_PULL_REQUEST variable to leave test result-related comments in a pull request. Redirecting this output might allow us to store results for non-pr-triggered builds.

Cloned by: https://github.com/jdi-testing/jdi-light/issues/2071
All required changes are available in following PR: https://github.com/jdi-testing/jdi-light/pull/2073

**Comment by clockworkbanana** [comment]
The changes were successfully merged into master.
1. master build is triggered weekly by cron
2. Build, triggered by cron or manually, executes tests using java 8--13 and writes the results to dedicated pull request (at this moment it is 2058, however this value can be changed)
3. Builds, triggered by PRs or Pushes would execute on single JDK version (openjdk8 right now)
